src/ng-generate/prompter/index.ts

```diff
@@ -18,26 +18,20 @@ import path from 'path';
 import inquirer from 'inquirer';
 import {lastValueFrom, Subscriber} from 'rxjs';
 import {TemplateHelper} from '../../utils/template-helper';
-import {Schema, ComponentType} from '../components/shared/schema';
+import {ComponentType, Schema} from '../components/shared/schema';
 
 import {loader, reorderAspectModelUrnToLoad, writeConfigAndExit} from './utils';
 import {virtualFs} from '@angular-devkit/core';
-import {anotherFile, configFileName, createOrImport, importConfigFile} from './prompts-questions/shared/prompt-simple-questions';
+import {
+    anotherFile,
+    configFileName,
+    createOrImport,
+    importConfigFile
+} from './prompts-questions/shared/prompt-simple-questions';
 import {tablePrompterQuestions} from './prompts-questions/table/prompt-questions';
-import {pathDecision, requestAspectModelUrnToLoad} from './prompts-questions/shared/prompt-complex-questions';
+import {pathDecision, requestAspectModelWithAspect} from './prompts-questions/shared/prompt-complex-questions';
 import {formPrompterQuestions} from './prompts-questions/form/prompt-questions';
 import {cardPrompterQuestions} from './prompts-questions/card/prompt-questions';
-//
-
-// import {ComponentType, Schema} from '../shared/schema';
-// import {generateCardComponent} from './generators/components/card';
-// import {generateExportCardDialog} from './generators/components/export-dialog/index';
-
-// export default function (cardSchema: CardSchema): Rule {
-//     return (tree: Tree, context: SchematicContext) => {
-//         generateComponent(context, cardSchema, ComponentType.CARD);
-//     };
-//
```

Comment:
If it's not going to be used, I think it's better to completely remove it.

Reply:
Its removed.